src/Language/PureScript/Ide/Types.hs

```diff
@@ -94,7 +94,41 @@ data IdeTypeOperator = IdeTypeOperator
   , _ideTypeOpKind          :: Maybe P.Kind
   } deriving (Show, Eq, Ord, Generic, NFData)
 
-makePrisms ''IdeDeclaration
+_IdeDeclValue :: Traversal' IdeDeclaration IdeValue
```

Comment:
Looks like we can write these as traversals rather than prisms without losing anything in our logic. Is that right?